integration-cli/docker_cli_kill_test.go

```diff
@@ -42,9 +42,11 @@ func (s *DockerSuite) TestKillDifferentUserContainer(c *check.C) {
 	dockerCmd(c, "kill", cleanedContainerID)
 
 	out, _ = dockerCmd(c, "ps", "-q")
-	if strings.Contains(out, cleanedContainerID) {
-		c.Fatal("killed container is still running")
-	}
+	processContainerID := strings.TrimSpace(out)
+
+	c.Assert(waitRun(processContainerID), check.IsNil)
```

Comment:
> `waitRun` will wait for the specified container to be running, maximum 5 seconds

So in that case, the containers has been already kill, it will return an error.

Anyway, it wasn't done before, it does not need to be done after the change. No need to add extra checks, you just need `c.Assert(out, checker.Not(checker.Contains), cleanedContainerID, check.Commentf("killed container is still running"))` to replace the `if …` :wink:.